forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/SendSubmissionToODSDelegate.java

```diff
@@ -82,9 +84,14 @@ private void sendSubmissionToODS(DelegateExecution execution) throws IOException
         if (managerGuid != null) {
             values.put("manager_guid", String.valueOf(managerGuid));
         }
-
         ObjectMapper objectMapper = new ObjectMapper();
-        String json = objectMapper.writeValueAsString(values);
+        String json = "";
+        if (objectKeycase.equals("snake_case")) {
```

Comment:
I'm not super familiar with these functions, but isn't this saying "if `objectKeycase` is `snake_case` then convert it to `snake_case`?

Reply:
Oh, is the `objectKeycase` the desired form for the variable?